app/views/external_users/claims/_form.html.haml

```diff
@@ -1,5 +1,5 @@
 #claim-form{ data: { claim_id: @claim.id } }
-  = form_for @claim, as: :claim, builder: AdpFormBuilder, html: { novalidate: true }, multipart: true, authenticity_token: true do |f|
+  = form_with model: @claim, scope: :claim, builder: GdsAdpFormBuilder, html: { novalidate: true }, multipart: true, authenticity_token: true do |f|
```

Comment:
(nit) I'm not 100% certain but I think the `scope: :claim` may be redundant here. I believe that `form_with` infers the scope from the model.

Reply:
Form submission errors with setting the scope,
`ActionController::ParameterMissing - param is missing or the value is empty`